homeassistant/components/rest/binary_sensor.py

```diff
@@ -114,16 +125,70 @@ def is_on(self):
         """Return true if the binary sensor is on."""
         return self._is_on
 
+    @property
+    def extra_state_attributes(self):
+        """Return the state attributes."""
+        return self._attributes
```

Comment:
You can replace this property with the use of `self._attr_extra_state_attributes` instead of `self._attributes`

Reply:
Done (throughout the class)